app/src/pages/common/modals/confirmModal.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { ModalLayout, withModal } from 'components/main/modal';
+import { hideModalAction, confirmModalAction } from 'controllers/modal';
+
+@withModal('confirmModal')
+@connect(null, {
+  confirmModal: confirmModalAction,
+  hideModal: hideModalAction,
+})
+export class ConfirmModal extends Component {
```

Comment:
I think `ConfirmationModal` sounds better (so no confusion between `confirmModal` as an action and `ConfirmModal` as a component)